tools/config/configure.R

```diff
@@ -6,7 +6,7 @@ candidates <- c("CXX11", "CXX1X", "CXX")
 for (candidate in candidates) {
    config <- read_r_config(candidate, envir = NULL)
    value <- config[[candidate]]
-   if (!is.null(value)) {
+   if (!is.null(value) && !(startsWith(value, "ERROR:"))) {
```

Comment:
Note that `startsWith()` is a relatively new addition to R, and we try to preserve compatibility with older versions of R when possible. I'd prefer a `grepl("^ERROR")` here, but really the fix should come in `read_r_config()` (the intention was that it would return `NULL` if the value didn't exist or an error occurred)

Reply:
Okay, I will give that a try and then force-push here.